src/Aspire.Cli/Backchannel/BackchannelDataTypes.cs

```diff
@@ -5,53 +5,6 @@
 
```

Comment:
If all that is left in this file is the data types for the "Extension" Backchannel, can we rename it to `ExtensionBackchannelDataTypes.cs` so there aren't 2 files with the same name?

Reply:
Yep, we can do that.